tests/legacy-cli/e2e/tests/build/material.ts

```diff
@@ -6,7 +6,7 @@ import { updateJsonFile } from '../../utils/project';
 const snapshots = require('../../ng-snapshot/package.json');
 
 export default async function () {
-  await ng('add', '@angular/material');
+  await ng('add', '@angular/material@next');
```

Comment:
We probably need a conditional here that checks the CLI version being tested for `next` so that when material's `@next` is at 11 these tests don't fail for the 10.x branch